After two years in hell...

batflarrow opened the pull request:
Obviously everybody's doing this for the t-shirt, but for me its just and added benefit.The workshop taught me something entirely new, and it was a fun experience. Please do read my file in _posts, it might amuse you.

Thanks;

rossoskull commented on _posts/2019-10-05-amit.md:
```diff
@@ -0,0 +1,28 @@
+github_id: "batflarrow"
+---
+
+Heya everyone, myself batflarrorw, the combination of Batman Flash and Arrow.After two years of hell in jee coaching, i finally reached here.I thought I'll live "Majegi ki Zindagi", but frankly its the same hell:(.We werent allowed to use mobiles and basically internet before, but now even though we have devices the internet is pretty much shit.Sala har mahine ek exam daaldete hain, kya pure saal ...
```
I really appreciate the comment 😄  But make sure it is only 1-2 lines, 10-20 words. And also remove the other lines which are not needed.

rossoskull reviewed (commented):
Please make the changes suggested so that I can review and merge it.

batflarrow committed:
Update 2019-10-05-amit.md

batflarrow committed:
After Two years in Hell

Obviously everybody's doing this for the t-shirt, but for me its just and added benefit.The workshop taught me something entirely new, and it was a fun experience. Please do read my file in _posts, it might amuse you.

Thanks;

batflarrow committed:
Delete 2019-10-05-amit.md

batflarrow committed:
After two years in hell...

batflarrow committed:
After two years in Hell

batflarrow committed:
After two years in Hell

rossoskull commented:
Close this PR and reopen a new one with only a single commit.

anshumanv commented:
> Obviously everybody's doing this for the t-shirt, but for me its just and added benefit.

Yes, the key takeaway is the experience and not the incentives, great! ✨
